### ISSUE TITLE
bank-webhook , updating balance issue for new user having zero balance

### ISSUE THREAD

**Issue by Akshat-Kumar-work** [100xdevs-cohort-2/week-18-2-ci-cd#60]
When the **new user** having **zero balance** tried to add balance, after the on-ramp-transaction success, the balance will not addedd to the user balance table, because we are only updating the balance , it will provide no record found error, hence we have to check if the balance array length is zero then we dont want to update the balance we want to create the balance record.
<img width="520" alt="bug" src="https://github.com/100xdevs-cohort-2/week-18-2-ci-cd/assets/104721125/4327b0d6-4237-46d1-aa77-e2f5fe4c9542">

**Comment by amarnathdhumal** [comment]
When creating new user in authOptions by prisma.user.create we can also add Balance : {create:  { amount: 0, locked: 0 } }this also will work.

**Comment by krishna4698** [comment]
yes we just have to do something that evenever new person signups we have to give them a balance of 0.. Just do some updations in file where you are signuping a new user just add a line  regarding above information
